src/components/AppNavHeader.tsx

```diff
@@ -22,7 +23,15 @@ export function AppNavHeader({
 }: NativeStackHeaderProps) {
   const { styles } = useStyles(defaultStyles);
   const { ChevronLeft } = useIcons();
-  const title = options.title || route.name;
+  const { headerTitle } = options;
+  const { headerString, headerFunction } =
+    typeof headerTitle === 'string'
```

Comment:
I think the type of `headerTitle` is `string | function` so you could probably reverse this and be able to remove the second ternary.

Reply:
👍 Good point